optimum/intel/openvino/quantization.py

```diff
@@ -694,6 +696,12 @@ def _prepare_visual_causal_lm_calibration_data(
         Currently, collects data only for a language model component.
         """
 
+        @lru_cache(maxsize=1)
+        def log_update_nncf_warning_message():
+            if is_nncf_version("<=", "2.18"):
+                # TODO (Nikita): Remove once NNCF 2.19 is released.
+                logger.warning("If you are facing RAM OOM issues, please update to the latest NNCF develop version.")
```

Comment:
you can use the log once utility in https://github.com/huggingface/optimum/blob/267500ead123103ff6d61d07d50166c326ace27a/optimum/utils/logging.py#L272

Reply:
Thanks, changed